src/views/CampusSafety/views/LostAndFound/index.tsx

```diff
@@ -362,54 +393,96 @@ const LostAndFound = () => {
 
       {titleCard()}
 
-      {/* Active Missing Item Reports */}
-      <Grid container justifyContent="center" spacing={3} marginTop={3}>
-        <Grid item xs={12} md={10}>
-          <Card>
-            <CardHeader
-              className="gc360_header"
-              title={
-                <Typography variant="h5" align="center">
-                  My Active <span className={styles.yellowText}>Lost</span> Item Reports
-                </Typography>
-              }
-            />
-          </Card>
+      {/* Recently Found Reports */}
+      {foundReports.length > 0 && (
+        <Grid container justifyContent="center" spacing={3} marginTop={3}>
+          <Grid item xs={12} md={10}>
+            <Card>
+              <CardHeader
+                className="gc360_header"
+                title={
+                  <Grid container alignItems="center" position="relative" justifyContent="center">
+                    {/* Badge positioned on the top-left corner */}
+                    <Badge
```

Comment:
The badge appears over the text on certain mobile screens sizes:
<img width="300" alt="Screen Shot 2024-12-04 at 19 04 50" src="https://github.com/user-attachments/assets/507fdbdf-69ca-4051-9108-848fb9367230">

Reply:
I resolved this in my commits